Reworing VNR API (#435)
xmos/fwk_voice: modules/lib_vnr/doc/src/getting_started.rst

The VNR estimator module is obtained as part of the parent ``fwk_voice`` repo clone. It is present in ``fwk_voice/modules/lib_vnr``

> Both feature extraction and the inference parts of ``lib_vnr`` can be compiled as a static library.

Reviewer: ```suggestion
Both the feature extraction and inference components of ``lib_vnr`` are compiled into a single static library.
```
